### PR TITLE
Log interactions

### LINKED ISSUE #43: Make the server url to send interactions to changeable

### PR DESCRIPTION
**Pull request by Kaleab470**
Closes #43

### CODE REVIEW

File: src/main/webapp/creatives/constants.js [hunk 1]
```diff
@@ -100,3 +100,14 @@ export const CreativeErrorCode = {
   MESSAGES_NOT_FOLLOWING_SPEC: 1110,
   PLAYER_RESPONSE_TIMEOUT: 1111,
 };
+
+/** A list of server routes that keep track of each interaction when fetched. */
+export const Urls = {
```
Comment (kristen-m) [inline src/main/webapp/creatives/constants.js:105]: Glad that moving this here worked!

File: src/main/webapp/creatives/maps/UserActivityLogger.js [hunk 2]
```diff
@@ -1,45 +1,58 @@
...
+export default class UserActivityLogger {
 
   constructor() {
     /**
      * Each instance gets a correlator to uniquely identify it.
      * @private @const {string}
      */
     this.correlator_ = generateSessionId();
+
+    /** @private @const {string} */
```
Comment (kristen-m) [inline src/main/webapp/creatives/maps/UserActivityLogger.js:14]: This variable might benefit from some description here but if you think baseUrl is descriptive enough then it's good as it

File: src/main/webapp/creatives/constants.js (hunk 1 above)
Comment (damargulis) [inline src/main/webapp/creatives/constants.js:105]: Make sure to also remove this from utils.js

Comment (Kaleab470) [reply src/main/webapp/creatives/constants.js:105]: Yeah that should be in my other PR

**Pull request merged**